cmd/plugin/cli/dogu-config/config.go

```diff
@@ -0,0 +1,33 @@
+package dogu_config
+
+import (
+	"fmt"
+	"github.com/cloudogu/kubectl-ces-plugin/cmd/plugin/cli/util"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"github.com/cloudogu/kubectl-ces-plugin/pkg/plugin/dogu-config"
+)
+
+const (
+	errMsgDoguConfigServiceCreate = "cannot create config service: %w"
+)
+
+type defaultServiceFactory struct {
+	cliConfig configTransporter
+}
+
+func (sf *defaultServiceFactory) create(doguName string) (doguConfigService, error) {
+	k8sArgs := sf.cliConfig.Get(util.CliTransportParamK8sArgs)
+	cfg := (k8sArgs).(*genericclioptions.ConfigFlags)
```

Comment:
🚫 **[checkstyle]** <forcetypeassert> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>type assertion must be checked